src/app/security/services/security.service.spec.ts

```diff
@@ -1,17 +1,19 @@
 import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
 import { AppConfig } from '../../app.config';
-import { TestBed } from '@angular/core/testing';
 import { SecurityService } from './security.service';
-import { AuthorizationHeaderName } from '../models/access-token';
 import { User } from '../models/user.model';
+import { SecurityContext } from './security-context.service';
+import { TestBed } from '@angular/core/testing';
+import { AuthorizationHeaderName } from '../models/access-token';
 
-let storageSpy, httpMock, securityService: SecurityService;
+let storageSpy, httpMock, securityService: SecurityService, securityContext: SecurityContext;
+const fakeUrl = 'https://doesnot.matter.com';
```

Comment:
do you know that this url actually exists?, could not you use something form platform.hmcts.net domain?